index.js

```diff
@@ -3,7 +3,8 @@
 const darksky = require("./apis/darksky").DarkSkyAPI,
 	weatherunderground = require("./apis/weatherunderground").WundergroundAPI,
 	openweathermap = require("./apis/openweathermap").OpenWeatherMapAPI,
-      	weewx = require("./apis/weewx").WeewxAPI,
+	weewx = require("./apis/weewx").WeewxAPI,
+	smartweather = require('./apis/smartweather').SmartWeatherAPI,
```

Comment:
IMHO this should become:
```suggestion
	tempest = require('./apis/weatherflow').TempestAPI,
```

Reply:
This is a much bigger change, but I'll take a look at it. I need to rename the API and rename files. I need to work out how to rename a file without loosing the change history.

Reply:
`git mv`

Reply:
Done!